api/v1beta1/module_types.go

```diff
@@ -81,6 +81,22 @@ type Build struct {
 	KanikoParams *KanikoParams `json:"kanikoParams,omitempty"`
 }
 
+type Sign struct {
+	// +optional
+	// Image to sign, ignored if a Build is present, required otherwise
+	UnsignedImage string `json:"unsignedImage,omitempty"`
+
+	// a secret containing the private key used to sign kernel modules for secureboot
+	KeySecret *v1.LocalObjectReference `json:"keySecret"`
+
+	// a secret containing the public key used to sign kernel modules for secureboot
+	CertSecret *v1.LocalObjectReference `json:"certSecret"`
+
+	// +optional
+	// paths inside the image for the kernel modules to sign (if ommited all kmods are signed)
+	FileList []string `json:"filesToSign,omitempty"`
```

Comment:
```suggestion
	FilesToSign []string `json:"filesToSign,omitempty"`
```
For consistency? or maybe the other way around?

Reply:
Do we really need this list? Can't we just sign all the .ko file under the appropriate directory (/usr/lib/modiles/<kernel>)?

Reply:
FilesToSign is clearer in the yaml than FileList so lets go with that.

For some reason I cant reply directly to your comment about having adding omitempty, I hadn't added it so it would match the attributes for the km.Build field. It makes sense though so I've force pushed a change with this and the FileList change.

Reply:
> Do we really need this list? Can't we just sign all the .ko files under the appropriate directory (/usr/lib/modiles/)?

Follow-on issue: #94. Let's merge this for now.